utils/fhir_api.py

```diff
@@ -0,0 +1,54 @@
+"""HAPI FHIR queries"""
+import json
+
+import requests
+import yaml
+
+
+def build_url(endpoint: str):
+    """
+    Dynamically building the searching query
+
+    Args:
+        endpoint (str): one of theendpoint available in the configuration file
+
+    Returns:
+        url (str): the searching query
+    """
+
+    with open("../config/settings.yml", "r", encoding="utf-8") as yaml_file:
+        config = yaml.safe_load(yaml_file)
+
+    url_base = config["fhir_api_base"]
+    url_endpoint = config["endpoints"].get(endpoint)
```

Comment:
I feel using `.get()` think we could benefit with most `dict` key calls make things more robust perhaps?
even raising an error in the case where the URLs are not correctly loaded?